engine/node_matlab.cc

```diff
@@ -493,6 +493,41 @@ namespace MathDAG
     checkArg(0,0);
     return o<<"frac("<<arguments[0][0]->matlab()<<")";
   }
+  
+  template <>
+  ostream& OperationDAG<OperationType::percent>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"100*("<<arguments[0][0]->matlab()<<")";
+  }
+ 
+  template <>
+  ostream& OperationDAG<OperationType::gamma>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"gamma("<<arguments[0][0]->matlab()<<")";
+  }
+  
+  template <>
+  ostream& OperationDAG<OperationType::digamma>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"psi(2,("<<arguments[0][0]->matlab()<<"))";
+  }      
+  
+  template <>
+  ostream& OperationDAG<OperationType::polygamma>::matlab(ostream& o) const
+  {
+    checkArg(0,0); checkArg(1,0);
+    return o<<"psi(("<<arguments[1][0]->matlab()<<"),("<<arguments[0][0]->matlab()<<"))";
```

Comment:
You would need the floor function here to ensure the order is an integer.

Reply:
Done, thanks :)